server/src/project/_utils/transform-milestones.ts

```diff
@@ -146,6 +146,21 @@ function applyDisplayDate(milestone, project) {
   if (milestone._dcp_milestone_value === '780593bb-ecc2-e811-8156-1458d04d0698') milestone.display_date = milestone.dcp_actualenddate;
   if (milestone._dcp_milestone_value === '483beec4-dad0-e711-8116-1458d04e2fb8') milestone.display_date = milestone.dcp_actualenddate;
   if (milestone._dcp_milestone_value === '4a3beec4-dad0-e711-8116-1458d04e2fb8') milestone.display_date = milestone.dcp_actualenddate;
+  
+  /*
+  * The following is for task 13239
+  * The date displayed should be the CRM Review Meeting date for certain milestones if the milestone is "in progress" or "completed", and the meeting date is <= current date
+  * I have included a check to make sure that the field is not null to make sure we are not overwriting dates with nothing
+  * 2 is completed, 717170000 is in progress
+  * Application Reviewed at City Planning Commission Review Session, City Planning Commission Vote - Need to get the ids for the other three milestones
+  */
+  if((milestone.statuscode === 2) || (milestone.statuscode === 717170000)) {
+    if ((milestone._dcp_milestone_value === '8e3beec4-dad0-e711-8116-1458d04e2fb8') || (milestone._dcp_milestone_value === 'a43beec4-dad0-e711-8116-1458d04e2fb8')) { 
+      if((milestone.dcp_reviewmeetingdate) && (Date.now() >= milestone.dcp_reviewmeetingdate.getTime())) {
+        milestone.display_date = milestone.dcp_reviewmeetingdate;
+      }
+    }
+  }
 
   return milestone;
 }
```

Comment:
I'm not sure the purpose of DisplayDate2, but in the templates the DisplayDate is conditionally displayed only if DisplayDate2 exists. Do you need to add this new logic to the `applySecondDisplayDate()` function as well?

Reply:
It looks to me like DisplayDate2 is only used if you want to show a date range, like "August 23, 2019 - September 6, 2019".  Otherwise it looks like it uses DisplayDate.  I don't think any modifications are necessary here.

Reply:
oh I just realized that for those two specific miltestones, display_date_2 will always be null